theme/themes/eea/views/card.overrides

```diff
@@ -451,3 +452,8 @@
 .ui.card .header > a {
   .useTitleMaxLines();
 }
+
+// Meta align fix
+.ui.card.u-card .meta:after {
+    content: none;
+}
```

Comment:
This override added to remove an empty ":after" that is added from listing-cards.less and breaks the metadata justify-content. But if you need that after's content, remove the override again to use it